src/components/home/Home.tsx

```diff
@@ -0,0 +1,8 @@
+/** Base component that renders the Home page */
+const Home = () => {
```

Comment:
This is just me, but I'd add content here

Reply:
That is upcoming in the next PR :). Just wanted to have these empty templates to test the routing (root to home page, etc.)